src/krate.rs

```diff
@@ -1140,6 +1164,8 @@ pub fn new(req: &mut Request) -> CargoResult<Response> {
         };
 
         let license_file = new_crate.license_file.as_ref().map(|s| &**s);
+        // Create a transaction on the database, if there are no errors,
+        // commit the transactions to record a new or updated crate.
```

Comment:
The transaction actually starts with the `conn.transaction` call [up here](https://github.com/rust-lang/crates.io/pull/936/files#diff-7ab30501521c288e1d8e1e994692b1a2L1129), I think this comment should be moved to go with that.